tests/e2e/rpmStatsFunctionalityLastUploadFilterAllRanges.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
       browser.assert.strictEqual(exists, true, 'exported rpm csv file exists ');
     } catch (error) {
       console.log(error);
+      browser.quit();
```

Comment:
**suggestion**: Create function from this try/catch block. You are using it on several places - use DRY here :)

Reply:
added function to module and imported to each test . see commit 5fe1caf6baee2d114c2f21b601ec622427cfaf80